docs/content/users/install/docker-installation.md

```diff
@@ -39,6 +39,12 @@ You’ll need a Docker provider on your system before you can [install DDEV](dde
 
     #### Migrating Projects from Docker Desktop to Colima
 
+    !!!tip "Docker Desktop may have left a bad config.json"
+        Remove the `credsStore` line in `~/.docker/config.json` if you have trouble.
+
+    1. Open Docker's `config.json` file (`~/.docker/config.json`)
+    2. Remove the `credsStore` line.
```

Comment:
I’m not sure why these steps repeat the exact same instruction in a numbered list format outside the admonition, so it seems best to remove them:

```suggestion
```